File: ClasesTienda/src/Modelos/Productos.java

package Modelos;

/**
 *
 * @author jdavi
 */
public class Productos {
    private int IdProducto;
    private String NombreProducto;
    private String DirIMGProd;
    private float Precio;
    private int Stock;
    private int IdCategoria;
    
    public Productos(int IdProducto, String NombreProducto, String DirIMGProd, float Precio, int Stock, int IdCategoria){
        this.IdProducto=IdProducto;
        this.NombreProducto=NombreProducto;
        this.DirIMGProd=DirIMGProd;
        this.Precio=Precio;
        this.Stock=Stock;
        this.IdCategoria=IdCategoria;
    }

    public int getIdProducto() {
        return IdProducto;
    }

    public void setIdProducto(int IdProducto) {
        this.IdProducto = IdProducto;
    }

    public String getNombreProducto() {
        return NombreProducto;
    }

    public void setNombreProducto(String NombreProducto) {
        this.NombreProducto = NombreProducto;
    }
    
    public String getDirIMGProd() {
        return DirIMGProd;
    }

    public void setDirIMGProd(String DirIMGProd) {
        this.DirIMGProd = DirIMGProd;
    }

    public float getPrecio() {
        return Precio;
    }

    public void setPrecio(float Precio) {
        this.Precio = Precio;
    }

    public int getStock() {
        return Stock;
    }

    public void setStock(int Stock) {
        this.Stock = Stock;
    }
    
    public int getIdCategoria() {
        return IdCategoria;
    }

    public void setIdCategoria(int IdCategoria) {
        this.IdCategoria = IdCategoria;
    }
    
}

File: ClasesProy/ClasesObj/Productos.php
<?php


class Productos {
    private $IdProducto;
    private $DirIMGProd;
    private $NombreProducto;
    private $Descripcion;
    private $Precio;
    private $Stock;
    private $IdCategoria;
    private $ActividadId;

    public function __construct($IdProducto=null,$NombreProducto=null,$DirIMGProd=null, $Descripcion=null,$Precio=null,$Stock=null,$IdCategoria=null,$ActividadId=1){
        $this->IdProducto = $IdProducto;
        $this->NombreProducto = $NombreProducto;
        $this->DirIMGProd = $DirIMGProd;
        $this->Descripcion = $Descripcion;
        $this->Precio = $Precio;
        $this->Stock = $Stock;
        $this->IdCategoria = $IdCategoria;
        $this->ActividadId = $ActividadId;
    }

    public function SetIdProducto($IdProducto){
        $this->IdProducto=$IdProducto;
    }

    public function GetIdProducto(){
        return $this->IdProducto;
    }
    
    public function SetNombreProducto($NombreProducto){
        $this->NombreProducto=$NombreProducto;
    }

    public function GetNombreProducto(){
        return $this->NombreProducto;
    }

    public function SetDirIMGProd($DirIMGProd){
        $this->DirIMGProd=$DirIMGProd;
    }

    public function GetDirIMGProd(){
        return $this->DirIMGProd;
    }

    public function SetDescripcion($Descripcion){
        $this->Descripcion=$Descripcion;
    }

    public function GetDescripcion(){
        return $this->Descripcion;
    }

    
    public function SetPrecio($Precio){
        $this->Precio=$Precio;
    }

    public function GetPrecio(){
        return $this->Precio;
    }
    
    public function SetStock($Stock){
        $this->Stock=$Stock;
    }

    public function GetStock(){
        return $this->Stock;
    }
    
    public function SetIdCategoria($IdCategoria){
        $this->IdCategoria=$IdCategoria;
    }

    public function GetIdCategoria(){
        return $this->IdCategoria;
    }
    
    public function SetActividadId($ActividadId){
        $this->ActividadId=$ActividadId;
    }

    public function GetActividadId(){
        return $this->ActividadId;
    }

}

?>

